src/screens/BB-BUS/DetailScreen.js

```diff
@@ -107,20 +107,16 @@ export const DetailScreen = ({ route }) => {
 
   const openWebScreen = useOpenWebScreen(headerTitle, undefined, rootRouteName);
 
-  const refreshTime = useRefreshTime(`BBBUS-service-${id}`, consts.REFRESH_INTERVALS.BB_BUS);
```

Comment:
> removed caching for details page to avoid problems due to pst id not being a unique key without areaId

what about using `BBBUS-service-${areaId}-${id}` here? and reverting `BB_BUS: NEVER` to `BB_BUS: ONCE_A_DAY` again in `REFRESH_INTERVALS` constants? would this help to be able to change less code and keep a bit of caching?

Reply:
the problem is the following, if cache is used
1. open details of service `S` for area `A`
2. open details of service `S` for area `B`
-> cache for service `S` for area `A` is overwritten by service `S` for area `B`
3. open details of service `S` for area `A` again
-> cache is used but contains details for area `B`

this happens because the cache only works based on ids of entries, but the id of service `S` is the same for area `A` and area `B`

regardless of that, I changed the caching back from `NEVER` to `ONCE_A_DAY`. for the index screen bb bus queries

Reply:
thanks, this helped me to understand it more clearly. so the query with `variables: { id, areaId }` is just using the `id` for caching? i somehow thought that a combination of variables will be used.

Reply:
it is not using query variables at all for caching. just the id of the response

Reply:
jo, i did some more research.
we can come back to it at some later point: https://www.apollographql.com/docs/react/v2/caching/cache-configuration/#assigning-unique-identifiers